### PR TITLE
Add R&D teams page

### PR DESCRIPTION
**Pull request by jwilander**
Wrote up a quick page detailing who's on which team in R&D

### CODE REVIEW

File: site/content/internal/rd-teams.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+* Dylan Haussermann - QA
+
+## Platform Team
+* George Goldberg - Eng Lead
+* Jesus Espino - Engineer
+* Martin Kraft - Engineer
+* Daniel Schalla - Engineer
+* Miguel de la Cruz - Engineer
+* Katie Wiersgalla - PM
+* Lindy Isherwood - QA
```
Comment (crspeller) [inline site/content/internal/rd-teams.md]:
```suggestion
* Lindy Isherwood - QA

## Security Team
* Christopher Speller - Engineer
* George Goldberg - Eng Lead
* Daniel Schalla - Engineer
* Harrison Healey - Engineer
```

Comment (jwilander) [reply site/content/internal/rd-teams.md]: Oh yeah good call

File: site/content/internal/rd-teams.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+* Katie Wiersgalla - PM
+
+## Extensibility Team
+
+* Lev Brouk - Eng Lead
+* Christopher Speller - Engineer
+* Jason Blais - PM
+* Dylan Haussermann - QA
+
+## Platform Team
```
Comment (gabrieljackson) [inline site/content/internal/rd-teams.md]: nit: newlines after group section headers aren't consistent

File: site/content/internal/rd-teams.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+* Dean Whillier - Engineer
+* Miguel Alatzar - Engineer
+* Eric Sethna - PM
+* Linda - QA Lead
+
+## Cloud Team
+
+* Joram Wilander - Eng Lead
+* Jesse Hallam - Engineer
+* Carlos Panato - Engineer/DevOps
```
Comment (cpanato) [inline site/content/internal/rd-teams.md]: Maybe Engineer/Infrastructure ? Devops is a culture and not a job position :)

File: site/content/internal/rd-teams.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+* Lindy Isherwood - QA
+
+## Security Team
+
+* Christopher Speller - Engineer
+* George Goldberg - Eng Lead
+* Daniel Schalla - Engineer
+* Harrison Healey - Engineer
+
+## UX Team
```
Comment (lfbrock) [inline site/content/internal/rd-teams.md]: I've just been speaking with people about trying out having UX designers embedded in dev teams as well. 

To start with, teams will be: 
- Platform: Mike 
- Extensibility: Asaad
- Apps: Andrew

And for Cloud team, Katie will pull from whoever she needs (likely Andrew for end user facing things, Mike for admin things).

File: site/content/internal/rd-teams.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+* Miguel de la Cruz - Engineer
+* Katie Wiersgalla - PM
+* Lindy Isherwood - QA
+
+## Security Team
+
+* Christopher Speller - Engineer
+* George Goldberg - Eng Lead
+* Daniel Schalla - Engineer
+* Harrison Healey - Engineer
```
Comment (lfbrock) [inline site/content/internal/rd-teams.md]: Katie has been helping out as SMT PM

**Pull request merged**